packages/common/core/src/lib/normalize-options.ts

```diff
@@ -39,7 +39,9 @@ export function normalizeOptions<TSchema extends BaseSchema>(
         projectDirectory,
     );
 
-    const projectName = projectDirectory.replace(/\//g, '-');
+    const projectName = projectDirectory
+        .replace(/\//g, '-')
+        .replace(/\\/g, '-');
```

Comment:
May be cleaner to use `path.sep` here

Reply:
updated, cheers.

Reply:
change didn't work as expected. Reverted back to double replace for now.